### PR TITLE
Eosio build

### PR DESCRIPTION
**Pull request by pacificcode**
New build procedure from branch eosio_build.
Removed:
eosio-build.sh
scripts/install_dependencies.sh

Added:
eosio_build.sh
scripts/eosio_build_darwin.sh
scripts/eosio_build_dep
scripts/eosio_build_fedora.sh
scripts/eosio_build_ubuntu.sh

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -606,6 +608,13 @@ cd build
 cmake -G "Unix Makefiles" -DCMAKE_INSTALL_PREFIX=.. -DLLVM_TARGETS_TO_BUILD= -DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD=WebAssembly -DCMAKE_BUILD_TYPE=Release ../
 make -j4 install
 ```
+Add `WASM_LLVM_CONFIG` and `LLVM_DIR` to your `.bash_profile`:
+
+```bash
+echo "export WASM_LLVM_CONFIG=~/wasm-compiler/llvm/bin/llvm-config" >> ~/.bash_profile
+echo "export LLVM_DIR=/usr/local/Cellar/llvm/4.0.1/lib/cmake/llvm" >> ~/.bash_profile
```
Comment (jgiszczak) [inline README.md]: Should be /usr/local/Cellar/llvm@4/4.0.1/lib/cmake/llvm

File: README.md [hunk 2]
```diff
@@ -689,3 +697,84 @@ echo "export WASM_LLVM_CONFIG=~/wasm-compiler/llvm/bin/llvm-config" >> ~/.bash_p
 echo "export LLVM_DIR=/usr/local/Cellar/llvm/4.0.1/lib/cmake/llvm" >> ~/.bash_profile
```
Comment (jgiszczak) [inline README.md]: Also needs to be llvm@4.

File: README.md [hunk 3]
```diff
@@ -689,3 +697,84 @@ echo "export WASM_LLVM_CONFIG=~/wasm-compiler/llvm/bin/llvm-config" >> ~/.bash_p
...
+
+```
+
+Install Boost 1.66:
+
+```bash
+cd ~
+curl -L https://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.bz2 > boost_1.66.0.tar.bz2
+tar xf boost_1.66.0.tar.bz2
+echo "export BOOST_ROOT=$HOME/boost_1_64_0" >> ~/.bash_profile
```
Comment (jgiszczak) [inline README.md]: Needs to be $HOME/boost_1_66_0

File: README.md [hunk 4]
```diff
@@ -59,87 +66,82 @@ The following instructions detail the process of getting the software, building
...
 ```
-Choose whether you will be building for a local testnet or for the public testnet and jump to the appropriate section below.  Clone the EOS repository recursively as described and run eosio-build.sh located in the root `eos` folder.
+Choose whether you will be building for a local testnet or for the public testnet and jump to the appropriate section below.  Clone the EOS repository recursively as described and run eosio_build.sh located in the root `eos` folder.
 
 :warning: **As of February 2018, `master` is under heavy development and is not suitable for experimentation.** :warning:
 
 We strongly recommend following the instructions for building the public testnet version for [Ubuntu](#autoubuntupublic) or [Mac OS X](#automacpublic). `master` is in pieces on the garage floor while we rebuild this hotrod. This notice will be removed when `master` is usable again. Your patience is appreciated.
 
 <a name="autoubuntulocal"></a>
-#### :no_entry: Clean install Ubuntu 16.10 for a local testnet :no_entry:
```
Comment (jgiszczak) [inline README.md]: Restore no entry emojis.

File: README.md [hunk 5]
```diff
@@ -59,87 +66,82 @@ The following instructions detail the process of getting the software, building
...
-git checkout dawn-2.x
-./eosio-build.sh ubuntu
+git checkout DAWN-2018-01-25
+./eosio_build.sh
 ```
 
 Now you can proceed to the next step - [Running a node on the public testnet](#publictestnet)
 
 <a name="automaclocal"></a>
-#### :no_entry: MacOS Sierra for a local testnet :no_entry:
```
Comment (jgiszczak) [inline README.md]: Restore no_entry emojis.

File: README.md (hunk 1 above)
Comment (jgiszczak) [inline README.md]: Should be /usr/local/Cellar/llvm@4/4.0.1/lib/cmake/llvm

File: README.md (hunk 2 above)
Comment (jgiszczak) [inline README.md]: Should also be llvm@4

File: README.md (hunk 5 above)
Comment (pacificcode) [reply README.md]: Note: Corey specifically requested these to be removed. I put them back as per your request.

**Pull request merged**